From fc05bb73b3cb3e281233f9b1eff39bbf2e49f631 Mon Sep 17 00:00:00 2001
From: Ariel Aviad <aaviad@habana.ai>
Date: Thu, 05 Jun 2025 11:02:13 +0300
Subject: [PATCH] [SW-228312] g3: bring up: update files to match habanalabs repo

Habanalabs repo update gaudi3 bring up, this
patch update the changes to embedded specs
of print function

Change-Id: I771b98f0759e51561f2cc6793de88f0d8336d47b
Signed-off-by: Ariel Aviad <aaviad@habana.ai>
---

diff --git a/gaudi3/bringup/gaudi3_bringup.c b/gaudi3/bringup/gaudi3_bringup.c
index 35fd20d..9b6fa09 100644
--- a/gaudi3/bringup/gaudi3_bringup.c
+++ b/gaudi3/bringup/gaudi3_bringup.c
@@ -864,7 +864,7 @@
 				GAUDI3_PLL_TIMEOUT_USEC);
 
 		if (rc) {
-			dev_err(hdev->dev, "Failed to get PLL %d lock, die 0 ()\n", pll_index);
+			hl_err(hdev, "Failed to get PLL %d lock, die 0 ()\n", pll_index);
 			return -EIO;
 		}
 	}
@@ -926,7 +926,7 @@
 				GAUDI3_PLL_TIMEOUT_USEC);
 
 		if (rc) {
-			dev_err(hdev->dev, "Failed to get PLL %d lock, die 1 (%d)\n",
+			hl_err(hdev, "Failed to get PLL %d lock, die 1 (%d)\n",
 						pll_index, rc);
 			return -EIO;
 		}
@@ -1195,7 +1195,7 @@
 	if (hdev->fw_components & FW_TYPE_PREBOOT_CPU)
 		return 0;
 
-	dev_dbg(hdev->dev, "PLL init\n");
+	hl_dbg(hdev, "PLL init\n");
 
 	rc = gaudi3_nominal_hbm_1800(hdev, gaudi3);
 	if (rc)
@@ -1286,7 +1286,7 @@
 			GAUDI3_PLL_TIMEOUT_USEC);
 
 	if (rc) {
-		dev_err(hdev->dev, "LTA (%#llx) still busy\n",
+		hl_err(hdev, "LTA (%#llx) still busy\n",
 				offset + CSLICE_MISC_OFFSET + mmCACHE_MISC_LTA_INIT_BUSY);
 		ctx->rc = -EIO;
 		return;
@@ -1540,7 +1540,7 @@
 	if ((wr_addr == 0) && (rd_addr == 0))
 		return;
 
-	dev_err(hdev->dev, "AXI drain %s event: read address %#llx, write address %#llx\n",
+	hl_err(hdev, "AXI drain %s event: read address %#llx, write address %#llx\n",
 						type, rd_addr, wr_addr);
 }
 
@@ -1550,7 +1550,7 @@
 
 	drain_indication = RREG32(mmD0_PCIE_WRAP_BASE + mmPCIE_WRAP_AXI_DRAIN_IND);
 	if (drain_indication == 0xFFFFFFFF) {
-		dev_err(hdev->dev, "PCI link error\n");
+		hl_err(hdev, "PCI link error\n");
 		*pci_link_error = true;
 		return 1;
 	}
@@ -1710,7 +1710,7 @@
 				timeout);
 
 		if (rc) {
-			dev_err(hdev->dev, "Failed to get D2D FSM indication DONE on DIE%u (%d)\n",
+			hl_err(hdev, "Failed to get D2D FSM indication DONE on DIE%u (%d)\n",
 						die, rc);
 			return -EIO;
 		}
@@ -1737,7 +1737,7 @@
 			WREG32(addr, orig_id);
 			cmp_id = RREG32(addr);
 			if (cmp_id != orig_id) {
-				dev_err(hdev->dev,
+				hl_err(hdev,
 					"D2D sanity failed: %x, rtr: %x, orig: %x, cmp: %x\n",
 						hdcore, rtr, orig_id, cmp_id);
 				return false;
@@ -1763,7 +1763,7 @@
 	if (gaudi3->hw_cap_initialized & HW_CAP_D2D)
 		return 0;
 
-	dev_dbg(hdev->dev, "D2D init\n");
+	hl_dbg(hdev, "D2D init\n");
 
 	/*
 	 * remove protection to allow master d2d_spi to configure
@@ -2212,7 +2212,7 @@
 		return;
 	}
 
-	dev_dbg(hdev->dev, "Initializing CBC [F/W configuration]\n");
+	hl_dbg(hdev, "Initializing CBC [F/W configuration]\n");
 
 	WREG32(mmD0_PMMU_CBC_BASE + mmCBC_SET_SCRAM_EN, 0x1);
 }
@@ -2256,7 +2256,7 @@
 	if ((gaudi3->hw_cap_pdma_initialized & HW_CAP_PDMA_MASK) == HW_CAP_PDMA_MASK)
 		return;
 
-	dev_dbg(hdev->dev, "Initializing PDMA [F/W configuration]\n");
+	hl_dbg(hdev, "Initializing PDMA [F/W configuration]\n");
 
 	for (i = 0 ; i < prop->pdma_grp_max ; i++) {
 		reg_base = gaudi3_pdma_grp_blocks_bases[i];
@@ -2343,7 +2343,7 @@
 	if ((gaudi3->hw_cap_initialized & HW_CAP_EDMA_MASK) == HW_CAP_EDMA_MASK)
 		return;
 
-	dev_dbg(hdev->dev, "Initializing EDMAs [F/W configuration]\n");
+	hl_dbg(hdev, "Initializing EDMAs [F/W configuration]\n");
 
 	gaudi3_iterate_edmas(hdev, &iter_ctx);
 }
@@ -2418,7 +2418,7 @@
 	if ((gaudi3->hw_cap_tpc_initialized & HW_CAP_TPC_MASK) == HW_CAP_TPC_MASK)
 		return;
 
-	dev_dbg(hdev->dev, "Initializing TPCs [F/W configuration]\n");
+	hl_dbg(hdev, "Initializing TPCs [F/W configuration]\n");
 
 	gaudi3_iterate_tpcs(hdev, &iter_ctx);
 }
@@ -2486,7 +2486,7 @@
 	if ((gaudi3->hw_cap_initialized & HW_CAP_MME_MASK) == HW_CAP_MME_MASK)
 		return;
 
-	dev_dbg(hdev->dev, "Initializing MMEs [F/W configuration]\n");
+	hl_dbg(hdev, "Initializing MMEs [F/W configuration]\n");
 
 	gaudi3_iterate_mmes(hdev, &iter_ctx);
 }
@@ -2525,7 +2525,7 @@
 	if ((gaudi3->hw_cap_initialized & HW_CAP_ROT_MASK) == HW_CAP_ROT_MASK)
 		return;
 
-	dev_dbg(hdev->dev, "Initializing ROTs [F/W configuration]\n");
+	hl_dbg(hdev, "Initializing ROTs [F/W configuration]\n");
 
 	gaudi3_iterate_rotators(hdev, &iter_ctx);
 }
@@ -2597,7 +2597,7 @@
 	if ((gaudi3->hw_cap_dec_initialized & HW_CAP_DEC_MASK) == HW_CAP_DEC_MASK)
 		return;
 
-	dev_dbg(hdev->dev, "Initializing DECs [F/W configuration]\n");
+	hl_dbg(hdev, "Initializing DECs [F/W configuration]\n");
 
 	gaudi3_iterate_decoders(hdev, &iter_ctx);
 }
@@ -2869,7 +2869,7 @@
 	if (gaudi3->hw_cap_initialized & HW_CAP_HMMU_MASK)
 		return;
 
-	dev_dbg(hdev->dev, "Initializing HBM MMU (FW init)\n");
+	hl_dbg(hdev, "Initializing HBM MMU (FW init)\n");
 
 	dtlb_init_data.cntrl_page_size = build_tlb_ctrl_page_size(hdev);
 	gaudi3_iterate_dtlbs(hdev, &ctx);
@@ -2883,7 +2883,7 @@
 	if (gaudi3->hw_cap_initialized & HW_CAP_PMMU)
 		return;
 
-	dev_dbg(hdev->dev, "Initializing PCI MMU (FW init)\n");
+	hl_dbg(hdev, "Initializing PCI MMU (FW init)\n");
 
 	/* init PMMU prefetch-cache to store only hop5 */
 	RMWREG32(mmD0_PMMU_HBW_STLB_BASE + mmPSTLB_MEM_CACHE_CONFIG, 0x20,
@@ -3106,7 +3106,7 @@
 			1000,
 			200000);
 	if (rc) {
-		dev_err(hdev->dev, "wait for PCIE OB outstanding timed out\n");
+		hl_err(hdev, "wait for PCIE OB outstanding timed out\n");
 		return rc;
 	}
 
@@ -3148,7 +3148,7 @@
 	/* generate reset in DIE0 */
 	WREG32(d0_reset_reg, 0x1);
 
-	dev_dbg(hdev->dev, "Driver issued %s reset command\n", hard_reset ? "HARD" : "SOFT");
+	hl_dbg(hdev, "Driver issued %s reset command\n", hard_reset ? "HARD" : "SOFT");
 }
 
 static u32 gaudi3_get_qm_sw_map_idx(u32 base)
@@ -3175,7 +3175,7 @@
 
 	irq = gaudi3_get_qm_sw_map_idx(qm_reg_base);
 	if (irq == ARRAY_SIZE(gaudi3_qm_irq_map_table)) {
-		dev_err(hdev->dev, "invalid qm sw base: 0x%x, first base: 0x%x\n", qm_reg_base,
+		hl_err(hdev, "invalid qm sw base: 0x%x, first base: 0x%x\n", qm_reg_base,
 						first_qm_reg_base);
 		return;
 	}
@@ -3497,7 +3497,7 @@
 	/* Handle PRSTN Aggr */
 	if (parc_block_idx == 10) {
 		idx = ffs(sts0 & sts0_prstn_mask) - 1;
-		dev_err(hdev->dev, "Received PRSTN_SPI[%u] in D%u_PARC_INT_AGGR\n",
+		hl_err(hdev, "Received PRSTN_SPI[%u] in D%u_PARC_INT_AGGR\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PSOC_EVENT])
@@ -3512,18 +3512,18 @@
 	/* Handle VM_ALARMA_COMB Aggr */
 	if (parc_block_idx == 18) {
 		idx = ffs(sts0 & sts0_vm_alarm_mask) - 1;
-		dev_err(hdev->dev, "Received VM_ALARMA_COMB_SPI[%u] in D%u_PARC_INT_AGGR\n",
+		hl_err(hdev, "Received VM_ALARMA_COMB_SPI[%u] in D%u_PARC_INT_AGGR\n",
 			idx, die);
 	}
 
 	/* Handle COMBINED Aggr */
 	if (parc_block_idx == 24) {
 		idx = 0;
-		dev_err(hdev->dev, "Received COMBINED_SPI[%u] in D%u_PARC_INT_AGGR\n",
+		hl_err(hdev, "Received COMBINED_SPI[%u] in D%u_PARC_INT_AGGR\n",
 			idx, die);
 
 		if (sts0 & BIT(6))
-			dev_err(hdev->dev, "PSOC AXI drain event\n");
+			hl_err(hdev, "PSOC AXI drain event\n");
 	}
 
 	WREG32(mmD0_PARC_INT_AGGR_UART_COMB_BASE +
@@ -3916,7 +3916,7 @@
 		}
 	}
 
-	dev_dbg(hdev->dev, "PCIE error response 0x%llX\n", (addr_hi << 32) + addr_lo);
+	hl_dbg(hdev, "PCIE error response 0x%llX\n", (addr_hi << 32) + addr_lo);
 }
 
 static void gaudi3_clear_xresp_block(struct hl_device *hdev, u64 base, bool is_read)
@@ -3980,7 +3980,7 @@
 				break;
 
 			default:
-				dev_err(hdev->dev, "SEI event %u does not have event clear flow\n",
+				hl_err(hdev, "SEI event %u does not have event clear flow\n",
 						idx);
 				break;
 			}
@@ -4044,7 +4044,7 @@
 	int rc;
 
 	if (die == 1) {
-		dev_err(hdev->dev, "PCIE events from DIE1 are not supported\n");
+		hl_err(hdev, "PCIE events from DIE1 are not supported\n");
 		return;
 	}
 
@@ -4122,7 +4122,7 @@
 		while (err_msk) {
 			/* In case new event raised */
 			if ((err_msk & 1) && !(intr_cause & 1))
-				dev_err(hdev->dev,
+				hl_err(hdev,
 					"PCIE SEI event %u raised after PCIE SEI handling\n",
 					err_idx);
 			err_idx++;
@@ -4713,7 +4713,7 @@
 	/* There are rotator blocks only in HD 1/3/4/6 */
 	if ((die == 0 && (hdcore == 0 || hdcore == 2)) ||
 			(die == 1 && (hdcore == 1 || hdcore == 3)))  {
-		dev_err(hdev->dev, "No rotator interrupts are expected for DIE%u_HD%u!\n",
+		hl_err(hdev, "No rotator interrupts are expected for DIE%u_HD%u!\n",
 			die, hdcore);
 		return;
 	}
@@ -5286,7 +5286,7 @@
 	/* There are EDMA blocks only in HD 1/3/4/6 */
 	if ((die == 0 && (hdcore == 0 || hdcore == 2)) ||
 			(die == 1 && (hdcore == 1 || hdcore == 3)))  {
-		dev_err(hdev->dev, "No EDMA interrupts are expected for DIE%u_HD%u!\n",
+		hl_err(hdev, "No EDMA interrupts are expected for DIE%u_HD%u!\n",
 			die, hdcore);
 		return;
 	}
@@ -5716,7 +5716,7 @@
 	/* Handle PCIE */
 	if (sts0 & sts0_pcie_mask0) {
 		idx = 1;
-		dev_err(hdev->dev, "Received PCIE_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PCIE_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PCIE_EVENT])
@@ -5729,7 +5729,7 @@
 
 	if (sts0 & sts0_pcie_mask1) {
 		idx = ffs(sts0 & sts0_pcie_mask1) - 3;
-		dev_err(hdev->dev, "Received PCIE_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PCIE_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PCIE_EVENT])
@@ -5742,7 +5742,7 @@
 
 	if (sts0 & sts0_pcie_mask2) {
 		idx = ffs(sts0 & sts0_pcie_mask2) - 3;
-		dev_err(hdev->dev, "Received PCIE_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PCIE_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PCIE_EVENT])
@@ -5756,7 +5756,7 @@
 	/* Handle NIC */
 	if (sts0 & sts0_nic_mask) {
 		idx = ffs(sts0 & sts0_nic_mask) - 22;
-		dev_err(hdev->dev, "Received NIC_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received NIC_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_NIC_EVENT])
@@ -5769,7 +5769,7 @@
 
 	if (sts1 & sts1_nic_mask) {
 		idx = 11;
-		dev_err(hdev->dev, "Received NIC_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received NIC_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_NIC_EVENT])
@@ -5783,7 +5783,7 @@
 	/* Handle NCH */
 	if (sts1 & sts1_nch_mask) {
 		idx = ffs(sts1 & sts1_nch_mask) - 2;
-		dev_err(hdev->dev, "Received NCH_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received NCH_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_NCH_EVENT])
@@ -5797,7 +5797,7 @@
 	/* Handle PMMU */
 	if (sts1 & sts1_pmmu_mask) {
 		idx = ffs(sts1 & sts1_pmmu_mask) - 4;
-		dev_err(hdev->dev, "Received PMMU_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PMMU_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PMMU_EVENT])
@@ -5811,7 +5811,7 @@
 	/* Handle TS */
 	if (sts1 & sts1_ts_mask) {
 		idx = ffs(sts1 & sts1_ts_mask) - 9;
-		dev_err(hdev->dev, "Received TS_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received TS_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_TS_EVENT])
@@ -5825,7 +5825,7 @@
 	/* Handle PDMA */
 	if (sts1 & sts1_pdma_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received PDMA_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PDMA_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PDMA_EVENT])
@@ -5839,7 +5839,7 @@
 	/* Handle D2D */
 	if (sts1 & sts1_d2d_mask) {
 		idx = ffs(sts1 & sts1_d2d_mask) - 18;
-		dev_err(hdev->dev, "Received D2D_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received D2D_SPI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_D2D_EVENT])
@@ -5898,7 +5898,7 @@
 	/* Handle MME */
 	if (sts0 & sts0_mme_mask) {
 		idx = ffs(sts0 & sts0_mme_mask) - 1;
-		dev_err(hdev->dev, "Received MME_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received MME_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_MME_EVENT]) {
@@ -5916,7 +5916,7 @@
 	/* Handle TPC */
 	if (sts0 & sts0_tpc_mask) {
 		idx = ffs(sts0 & sts0_tpc_mask) - 16;
-		dev_err(hdev->dev, "Received TPC_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received TPC_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_TPC_EVENT]) {
@@ -5933,7 +5933,7 @@
 
 	if (sts1 & sts1_tpc_mask) {
 		idx = 17;
-		dev_err(hdev->dev, "Received TPC_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received TPC_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_TPC_EVENT]) {
@@ -5951,7 +5951,7 @@
 	/* Handle ROT */
 	if (sts1 & sts1_rot_mask) {
 		idx = ffs(sts1 & sts1_rot_mask) - 2;
-		dev_err(hdev->dev, "Received ROT_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received ROT_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_ROT_EVENT]) {
@@ -5969,7 +5969,7 @@
 	/* Handle Cache */
 	if (sts1 & sts1_cs_mask) {
 		idx = ffs(sts1 & sts1_cs_mask) - 6;
-		dev_err(hdev->dev, "Received CS_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received CS_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_CS_EVENT]) {
@@ -5987,7 +5987,7 @@
 	/* Handle STLB */
 	if (sts1 & sts1_stlb_mask) {
 		idx = ffs(sts1 & sts1_stlb_mask) - 22;
-		dev_err(hdev->dev, "Received STLB_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received STLB_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_STLB_EVENT]) {
@@ -6005,7 +6005,7 @@
 	/* Handle EDMA */
 	if (sts1 & sts1_edma_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received EDMA_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received EDMA_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_EDMA_EVENT]) {
@@ -6023,7 +6023,7 @@
 	/* Handle SOB */
 	if (sts1 & sts1_sob_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received SOB_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received SOB_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_SOB_EVENT]) {
@@ -6041,7 +6041,7 @@
 	/* Handle Decoder */
 	if (sts2 & sts2_dec_mask) {
 		idx = ffs(sts2 & sts2_dec_mask) - 3;
-		dev_err(hdev->dev, "Received DEC_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received DEC_SPI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_DEC_EVENT]) {
@@ -6100,7 +6100,7 @@
 	/* Handle MME */
 	if (sts0 & sts0_mme_mask) {
 		idx = ffs(sts0 & sts0_mme_mask) - 1;
-		dev_err(hdev->dev, "Received MME_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received MME_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_MME_EVENT]) {
@@ -6118,7 +6118,7 @@
 	/* Handle TPC */
 	if (sts0 & sts0_tpc_mask) {
 		idx = ffs(sts0 & sts0_tpc_mask) - 13;
-		dev_err(hdev->dev, "Received TPC_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received TPC_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_TPC_EVENT]) {
@@ -6136,7 +6136,7 @@
 	/* Handle ROT */
 	if (sts0 & sts0_rot_mask) {
 		idx = ffs(sts0 & sts0_rot_mask) - 22;
-		dev_err(hdev->dev, "Received ROT_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received ROT_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_ROT_EVENT]) {
@@ -6154,7 +6154,7 @@
 	/* Handle Cache */
 	if (sts0 & sts0_cs_mask) {
 		idx = ffs(sts0 & sts0_cs_mask) - 24;
-		dev_err(hdev->dev, "Received CS_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received CS_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_CS_EVENT]) {
@@ -6172,7 +6172,7 @@
 	/* Handle STLB */
 	if (sts0 & sts0_stlb_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received STLB_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received STLB_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_STLB_EVENT]) {
@@ -6190,7 +6190,7 @@
 	/* Handle EDMA */
 	if (sts1 & sts1_edma_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received EDMA_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received EDMA_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_EDMA_EVENT]) {
@@ -6208,7 +6208,7 @@
 	/* Handle HBM */
 	if (sts1 & sts1_hbm_mask) {
 		idx = ffs(sts1 & sts1_hbm_mask) - 2;
-		dev_err(hdev->dev, "Received HBM_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received HBM_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_HBM_EVENT]) {
@@ -6226,7 +6226,7 @@
 	/* Handle SOB */
 	if (sts1 & sts1_sob_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received SOB_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received SOB_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_SOB_EVENT]) {
@@ -6244,7 +6244,7 @@
 	/* Handle ARCFARM */
 	if (sts1 & sts1_arcfarm_mask) {
 		idx = 0;
-		dev_err(hdev->dev,
+		hl_err(hdev,
 			"Received ARC_FARM_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
@@ -6263,7 +6263,7 @@
 	/* Handle DUP */
 	if (sts1 & sts1_dup_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received DUP_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received DUP_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_DUP_EVENT]) {
@@ -6281,7 +6281,7 @@
 	/* Handle DEC */
 	if (sts1 & sts1_dec_mask) {
 		idx = ffs(sts1 & sts1_dec_mask) - 13;
-		dev_err(hdev->dev, "Received DEC_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received DEC_SEI[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_DEC_EVENT]) {
@@ -6342,7 +6342,7 @@
 	/* Handle CPU */
 	if (sts0 & sts0_cpu_mask) {
 		idx = ffs(sts0 & sts0_cpu_mask) - 1;
-		dev_err(hdev->dev, "Received CPU_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received CPU_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_CPU_EVENT])
@@ -6356,7 +6356,7 @@
 	/* Handle PCIE */
 	if (sts0 & sts0_pcie_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received PCIE_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PCIE_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PCIE_EVENT])
@@ -6370,7 +6370,7 @@
 	/* Handle NIC */
 	if (sts0 & sts0_nic_mask) {
 		idx = ffs(sts0 & sts0_nic_mask) - 6;
-		dev_err(hdev->dev, "Received NIC_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received NIC_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_NIC_EVENT])
@@ -6384,7 +6384,7 @@
 	/* Handle NCH */
 	if (sts0 & sts0_nch_mask) {
 		idx = ffs(sts0 & sts0_nch_mask) - 12;
-		dev_err(hdev->dev, "Received NCH_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received NCH_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_NCH_EVENT])
@@ -6398,7 +6398,7 @@
 	/* Handle PMMU */
 	if (sts0 & sts0_pmmu_mask) {
 		idx = 1;
-		dev_err(hdev->dev, "Received PMMU_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PMMU_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PMMU_EVENT])
@@ -6412,7 +6412,7 @@
 	/* Handle VM */
 	if (sts0 & sts0_vm_mask) {
 		idx = ffs(sts0 & sts0_vm_mask) - 16;
-		dev_err(hdev->dev, "Received VM_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received VM_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_VM_EVENT])
@@ -6426,7 +6426,7 @@
 	/* Handle PDMA */
 	if (sts0 & sts0_pdma_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received PDMA_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PDMA_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PDMA_EVENT])
@@ -6440,7 +6440,7 @@
 	/* Handle PSOC */
 	if (sts0 & sts0_psoc_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received PSOC_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PSOC_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PSOC_EVENT])
@@ -6454,7 +6454,7 @@
 	/* Handle PARC */
 	if (sts0 & sts0_parc_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received PARC_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PARC_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PARC_EVENT])
@@ -6468,7 +6468,7 @@
 	/* Handle D2D */
 	if (sts0 & sts0_d2d_mask) {
 		idx = ffs(sts0 & sts0_d2d_mask) - 27;
-		dev_err(hdev->dev, "Received D2D_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received D2D_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_D2D_EVENT])
@@ -6482,7 +6482,7 @@
 	/* Handle GLINK */
 	if (sts0 & sts0_glink_mask) {
 		idx = ffs(sts0 & sts0_glink_mask) - 29;
-		dev_err(hdev->dev, "Received GLINK_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received GLINK_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_GLINK_EVENT])
@@ -6495,7 +6495,7 @@
 
 	if (sts1 & sts1_glink_mask) {
 		idx = (ffs(sts0 & sts1_glink_mask)) + 3; /* +3 to distinguish name from sts0 */
-		dev_err(hdev->dev, "Received GLINK_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received GLINK_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_GLINK_EVENT])
@@ -6509,7 +6509,7 @@
 	/* Handle PLL */
 	if (sts1 & sts1_pll_mask) {
 		idx = ffs(sts0 & sts1_pll_mask) - 7;
-		dev_err(hdev->dev, "Received PLL_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PLL_SEI[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PLL_EVENT])
@@ -6611,7 +6611,7 @@
 	/* Handle MME */
 	if (sts0 & sts0_mme_mask) {
 		idx = ffs(sts0 & sts0_mme_mask) - 1;
-		dev_err(hdev->dev, "Received MME_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received MME_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_MME_EVENT]) {
@@ -6629,7 +6629,7 @@
 	/* Handle TPC */
 	if (sts0 & sts0_tpc_mask) {
 		idx = ffs(sts0 & sts0_tpc_mask) - 12;
-		dev_err(hdev->dev, "Received TPC_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received TPC_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_TPC_EVENT]) {
@@ -6647,7 +6647,7 @@
 	/* Handle ROT */
 	if (sts0 & sts0_rot_mask) {
 		idx = ffs(sts0 & sts0_rot_mask) - 21;
-		dev_err(hdev->dev, "Received ROT_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received ROT_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_ROT_EVENT]) {
@@ -6665,7 +6665,7 @@
 	/* Handle CS */
 	if (sts0 & sts0_cs_mask) {
 		idx = ffs(sts0 & sts0_cs_mask) - 23;
-		dev_err(hdev->dev, "Received CS_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received CS_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_CS_EVENT]) {
@@ -6683,7 +6683,7 @@
 	/* Handle STLB */
 	if (sts0 & sts0_stlb_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received STLB_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received STLB_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_STLB_EVENT]) {
@@ -6701,7 +6701,7 @@
 	/* Handle RTR */
 	if (sts0 & sts0_rtr_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received RTR_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received RTR_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_RTR_EVENT]) {
@@ -6718,7 +6718,7 @@
 
 	if (sts1 & sts1_rtr_mask) {
 		idx = ffs(sts1 & sts1_rtr_mask);
-		dev_err(hdev->dev, "Received RTR_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received RTR_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_RTR_EVENT]) {
@@ -6736,7 +6736,7 @@
 	/* Handle EDMA */
 	if (sts1 & sts1_edma_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received EDMA_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received EDMA_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_EDMA_EVENT]) {
@@ -6754,7 +6754,7 @@
 	/* Handle HBM */
 	if (sts1 & sts1_hbm_mask) {
 		idx = ffs(sts0 & sts1_hbm_mask) - 9;
-		dev_err(hdev->dev, "Received HBM_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received HBM_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_HBM_EVENT]) {
@@ -6772,7 +6772,7 @@
 	/* Handle SOB */
 	if (sts1 & sts1_sob_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received SOB_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received SOB_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_SOB_EVENT]) {
@@ -6790,7 +6790,7 @@
 	/* Handle ARCFARM */
 	if (sts1 & sts1_arcfarm_mask) {
 		idx = 0;
-		dev_err(hdev->dev,
+		hl_err(hdev,
 			"Received ARC_FARM_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
@@ -6809,7 +6809,7 @@
 	/* Handle DEC */
 	if (sts1 & sts1_dec_mask) {
 		idx = ffs(sts0 & sts1_dec_mask) - 15;
-		dev_err(hdev->dev, "Received DEC_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received DEC_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_DEC_EVENT]) {
@@ -6827,7 +6827,7 @@
 	/* Handle HIF */
 	if (sts1 & sts1_hif_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received HIF_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
+		hl_err(hdev, "Received HIF_DERR[%u] interrupt in D%u_CPU_INT_AGG_HDCORE%u\n",
 			idx, die, aggr_hdcore);
 
 		if (hdcore_handle_and_clear[HDCORE_HIF_EVENT]) {
@@ -6876,7 +6876,7 @@
 	/* Handle CPU */
 	if (sts0 & sts0_cpu_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received CPU_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received CPU_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_CPU_EVENT])
@@ -6890,7 +6890,7 @@
 	/* Handle PCIE */
 	if (sts0 & sts0_pcie_mask) {
 		idx = ffs(sts0 & sts0_pcie_mask) - 2;
-		dev_err(hdev->dev, "Received PCIE_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PCIE_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PCIE_EVENT])
@@ -6904,7 +6904,7 @@
 	/* Handle NIC */
 	if (sts0 & sts0_nic_mask) {
 		idx = ffs(sts0 & sts0_nic_mask) - 5;
-		dev_err(hdev->dev, "Received NIC_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received NIC_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_NIC_EVENT])
@@ -6918,7 +6918,7 @@
 	/* Handle NCH */
 	if (sts0 & sts0_nch_mask) {
 		idx = ffs(sts0 & sts0_nch_mask) - 11;
-		dev_err(hdev->dev, "Received NCH_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received NCH_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_NCH_EVENT])
@@ -6932,7 +6932,7 @@
 	/* Handle PMMU */
 	if (sts0 & sts0_pmmu_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received PMMU_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PMMU_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PMMU_EVENT])
@@ -6946,7 +6946,7 @@
 	/* Handle PDMA */
 	if (sts0 & sts0_pdma_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received PDMA_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PDMA_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PDMA_EVENT])
@@ -6960,7 +6960,7 @@
 	/* Handle PARC */
 	if (sts0 & sts0_parc_mask) {
 		idx = 0;
-		dev_err(hdev->dev, "Received PARC_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received PARC_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_PARC_EVENT])
@@ -6974,7 +6974,7 @@
 	/* Handle D2D */
 	if (sts0 & sts0_d2d_mask) {
 		idx = ffs(sts0 & sts0_d2d_mask) - 17;
-		dev_err(hdev->dev, "Received D2D_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received D2D_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_D2D_EVENT])
@@ -6988,7 +6988,7 @@
 	/* Handle RTR */
 	if (sts0 & sts0_rtr_mask) {
 		idx = ffs(sts0 & sts0_rtr_mask) - 19;
-		dev_err(hdev->dev, "Received RTR_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
+		hl_err(hdev, "Received RTR_DERR[%u] interrupt in D%u_CPU_INT_AGG_SHARED\n",
 			idx, die);
 
 		if (shared_handle_and_clear[SHARED_RTR_EVENT])
@@ -7090,32 +7090,32 @@
 
 	switch (qm_info->comp) {
 	case INT_COMP_TYPE_MME:
-		dev_err(hdev->dev, "Received QM SW event for D%u_HD%u_MME\n",
+		hl_err(hdev, "Received QM SW event for D%u_HD%u_MME\n",
 			qm_info->die, hd_in_die);
 		qm_data = &eq_dynamic_entry->mme_sei_data.control_data.qm_data;
 		eq_dynamic_entry->mme_sei_data.type = MME_DATA_TYPE_CTRL;
 		eq_dynamic_entry->hdr.size = cpu_to_le16(sizeof(struct hl_eq_mme_sei_data));
 		break;
 	case INT_COMP_TYPE_TPC:
-		dev_err(hdev->dev, "Received QM SW event for D%u_HD%u_TPC%u\n",
+		hl_err(hdev, "Received QM SW event for D%u_HD%u_TPC%u\n",
 			qm_info->die, hd_in_die, qm_info->instance);
 		qm_data = &eq_dynamic_entry->tpc_sei_data.qm_data;
 		eq_dynamic_entry->hdr.size = cpu_to_le16(sizeof(struct hl_eq_tpc_sei_data));
 		break;
 	case INT_COMP_TYPE_ROT:
-		dev_err(hdev->dev, "Received QM SW event for D%u_HD%u_ROT%u\n",
+		hl_err(hdev, "Received QM SW event for D%u_HD%u_ROT%u\n",
 			qm_info->die, hd_in_die, qm_info->instance);
 		qm_data = &eq_dynamic_entry->rot_sei_data.qm_data;
 		eq_dynamic_entry->hdr.size = cpu_to_le16(sizeof(struct hl_eq_rot_sei_data));
 		break;
 	case INT_COMP_TYPE_EDMA:
-		dev_err(hdev->dev, "Received QM SW event for D%u_HD%u_EDMA%u\n",
+		hl_err(hdev, "Received QM SW event for D%u_HD%u_EDMA%u\n",
 			qm_info->die, hd_in_die, qm_info->instance);
 		qm_data = &eq_dynamic_entry->edma_sei_data.qm_data[qm_info->instance];
 		eq_dynamic_entry->hdr.size = cpu_to_le16(sizeof(struct hl_eq_edma_sei_data));
 		break;
 	default:
-		dev_err(hdev->dev, "Received QM SW event with invalid component %u\n",
+		hl_err(hdev, "Received QM SW event with invalid component %u\n",
 			qm_info->comp);
 		return;
 	}
@@ -7228,7 +7228,7 @@
 	 * ref div must be set to 0x1, cannot be 0x0
 	 */
 	if (!refdiv) {
-		dev_err(hdev->dev, "refdiv = 0x0 for pll base_address: 0x%llx\n", pll_base_address);
+		hl_err(hdev, "refdiv = 0x0 for pll base_address: 0x%llx\n", pll_base_address);
 		return -EINVAL;
 	}
 
@@ -7541,7 +7541,7 @@
 		.data = &rr_config,
 	};
 
-	dev_dbg(hdev->dev, "Configure privileged LBW/HBW RRs\n");
+	hl_dbg(hdev, "Configure privileged LBW/HBW RRs\n");
 
 	gaudi3_iterate_rtr_ctrls(hdev, &ctx);
 }
@@ -7557,7 +7557,7 @@
 
 	rc = hl_init_pb_security(hdev, true);
 	if (rc) {
-		dev_err(hdev->dev, "Configuring privileged PBs failed!\n");
+		hl_err(hdev, "Configuring privileged PBs failed!\n");
 		return rc;
 	}
 
diff --git a/gaudi3/bringup/hbl_cn/gaudi3_cn_bringup.c b/gaudi3/bringup/hbl_cn/gaudi3_cn_bringup.c
index 859daa4..9097ade 100644
--- a/gaudi3/bringup/hbl_cn/gaudi3_cn_bringup.c
+++ b/gaudi3/bringup/hbl_cn/gaudi3_cn_bringup.c
@@ -94,7 +94,7 @@
 	}
 
 	if (rc)
-		dev_err(hdev->dev,
+		hbl_err(hdev,
 			"Failed to disable Rx WQE idx mismatch checker, port %d, rc %d\n", port,
 			rc);
 
@@ -842,7 +842,7 @@
 
 		NIC_WREG32(D0_NIC0_PHY_BASE + PRT_PHY_PHY_CFG_ACCESS, data);
 
-		dev_dbg(hdev->dev, "HL: writing reg: [0x%x] -> [0x%x] = 0x%x\n", addr, addr_remap,
+		hbl_dbg(hdev, "HL: writing reg: [0x%x] -> [0x%x] = 0x%x\n", addr, addr_remap,
 			data);
 	}
 }
@@ -858,7 +858,7 @@
 
 	file = filp_open(FW_HEX_PATH, O_RDONLY, 0);
 	if (IS_ERR(file)) {
-		dev_err(hdev->dev, "PHY FW HEX Path %s is invalid, cannot load FW.\n",
+		hbl_err(hdev, "PHY FW HEX Path %s is invalid, cannot load FW.\n",
 			FW_HEX_PATH);
 		return -EINVAL;
 	}
@@ -895,7 +895,7 @@
 		if (sscanf(buffer, "%x,%x", &addr, &data) == 2) {
 			fw_write_all(hdev, addr, data);
 		} else {
-			dev_err(hdev->dev, "PHY FW HEX %s parse error, line %s is not valid.\n",
+			hbl_err(hdev, "PHY FW HEX %s parse error, line %s is not valid.\n",
 				FW_HEX_PATH, buffer);
 
 			rc = -EINVAL;
@@ -910,7 +910,7 @@
 	fw_write_all(hdev, SRAM0_CFG_ADDR, 0x3);
 
 	if (line_count == MAX_HEX_LINES_TO_READ) {
-		dev_err(hdev->dev, "PHY FW HEX %s is too long, parsing stopped.\n", FW_HEX_PATH);
+		hbl_err(hdev, "PHY FW HEX %s is too long, parsing stopped.\n", FW_HEX_PATH);
 		rc = -EINVAL;
 	}
 
